### PR TITLE
feat: upgrade all dependencies in the package.json of figma plugin

### PR DESCRIPTION
**Pull request by 2020Deeya**
# Description

**Updating Dependencies and Verifying Build**
_Steps to Update Dependencies_

1] Update Dependencies Manually:

- Open the package.json file located at the root of the repository.
- Update all dependencies to their latest versions as per the npm registry.

2] Install Latest Dependencies:

- Run the following command to install the updated dependencies locally:
`yarn`

3] Verify Build Process:

- After updating and installing dependencies, ensure the build process works correctly by running:
`yarn build`
Screenshot 2024-08-01 at 4 00 33 PM

4] Verify Test Process:

- After updating and installing dependencies, ensure the test process works correctly by running:
`yarn test`
Screenshot 2024-08-01 at 3 59 18 PM

### CODE REVIEW

File: packages/@momentum-design/figma-plugin-assets-export/package.json [hunk 1]
```diff
@@ -31,32 +31,33 @@
...
   "dependencies": {
-    "@octokit/core": "^4.1.0",
-    "classnames": "^2.3.2",
+    "@octokit/core": "^6.1.2",
+    "classnames": "^2.5.1",
     "isomorphic-fetch": "^3.0.0",
-    "octokit-plugin-create-pull-request": "^4.0.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "octokit-plugin-create-pull-request": "^6.0.0",
```
Comment (arjunkesava) [inline packages/@momentum-design/figma-plugin-assets-export/package.json:59]: Please run figma plugin locally and raise a PR for icons / brand visuals library. To make sure that this upgrade is fine.

**Review by arjunkesava — COMMENTED** [review]
It would better to create a branch on your fork instead of building up on main upstream branch.

File: packages/@momentum-design/figma-plugin-assets-export/package.json (hunk 1 above)
Comment (2020Deeya) [reply packages/@momentum-design/figma-plugin-assets-export/package.json:59]: 1] Figma plugin is working fine on local. 
2] PR for icons and brand-visuals library are already created. Icons PR is ready to be merged after this. 
https://github.com/momentum-design/momentum-design/pull/703
https://github.com/momentum-design/momentum-design/pull/707

**Comment by 2020Deeya** [comment]
> It would better to create a branch on your fork instead of building up on main upstream branch.

As already discussed, the PR was already created before I had access to confluence for contributing guide. This will be taken care from the next bunch of PRs I create.

**Comment by 2020Deeya** [comment]
Related forked PR has been merged into main. Hence, closing this.

**Pull request closed without merging**